backend/app.server.js

```diff
@@ -57,21 +57,55 @@ app.get("/", (req, res) => {
   res.render("index");
 });
 
+// routes
+app.get("/new", (req, res) => {
+
+  const newRoom = {
+    "id": req.query.roomId,
+    "name": req.query.roomName,
+    "disableMeeting": false
+  }
+
+  let found = app.locals.roomsDetail.find(element => element.id == req.query.roomId);
+
+  if(!found){
+      app.locals.roomsDetail.splice( 1, 0, newRoom);
+  }
+
+  res.redirect(`/office#${req.query.roomId}`);
+});
+
+
+app.get("/remove", (req, res) => {
+
+
+  app.locals.roomsDetail = app.locals.roomsDetail.filter(function(value, index, arr){
+
+    return value.id !=req.query.roomId;
```

Comment:
I think this way is possible to remove static rooms.
Maybe temp rooms should has a flag that indicates the room is temporary, and filter by this flag.

Reply:
Hummmm good point of view... I'm going to fix this.

Reply:
Done!